### ISSUE TITLE
Proxy.revocable is undefined

### ISSUE THREAD

**Issue by melusc** [Tampermonkey/tampermonkey#1290]
`unsafeWindow.Proxy` works just like it should, it has `revocable`.
`window.Proxy` works fine except that it doesn't have `revocable`.

## Expected Behavior

```js
typeof Proxy.revocable === 'function' // true
'revocable' in Proxy // true
```

## Actual Behavior

```js
typeof Proxy.revocable === 'function' // false
'revocable' in Proxy // false
```

## Specifications

Same behaviour on

- Firefox: 91.0b3
- TM: 4.13.6136
- OS: Windows 10

and

- Edge: 91.0.864.70
- TM: 4.13
- OS: Windows 10

## Script

```js
// ==UserScript==
// @name	Proxy vs unsafeWindow.Proxy
// @match	*
// ==/UserScript==

'use strict';

// This works
const revocable = unsafeWindow.Proxy.revocable(
	{},
	{
		get: (target, name) => `[[${name}]]`,
	},
);
const proxy = revocable.proxy;
console.log(proxy.foo); // "[[foo]]"
revocable.revoke();

// This throws because .revocable is undefined
const revocableThrows = Proxy.revocable(
	{},
	{
		get: (target, name) => `[[${name}]]`,
	},
);
```

**Comment by derjanb** [comment]
Should be fixed at 4.14.6142

**Comment by melusc** [comment]
Yeah, works perfectly. Thank you.
